cmd/fs-v1-helpers_test.go

```diff
@@ -265,48 +265,121 @@ func TestFSDeletes(t *testing.T) {
 	// Seek back.
 	reader.Seek(0, 0)
```

Comment:
We may want to use io.SeekStart  (0 is a magic constant)
It's not directly in your PR but maybe you can patch it, too?